DependencyInjection/Configuration.php

```diff
@@ -110,7 +110,20 @@ public function getConfigTreeBuilder()
                 ->arrayNode('body_converter')
                     ->canBeEnabled()
                     ->children()
-                        ->scalarNode('validate')->defaultFalse()->end()
+                        ->scalarNode('validate')
+                            ->defaultFalse()
+                            ->beforeNormalization()
+                                ->ifTrue()
+                                ->then(function() {
+                                    if (!class_exists('Symfony\Component\OptionsResolver\OptionsResolver')) {
+                                        throw new InvalidConfigurationException(
```

Comment:
please just place this on a single line

Reply:
Ok but the line is going to be very long.
[There MUST NOT be a hard limit on line length; the soft limit MUST be 120
characters; lines SHOULD be 80 characters or less.](https://www.php-fig.org/psr/psr-2/)

Reply:
We consider exception messages as exceptions :)